### DISCUSSION TITLE
Multi Palette filter

### DISCUSSION THREAD

**Discussion by F-Burning** [opentoonz/opentoonz#6502] [Ideas]
### Is your feature request related to a problem? Please describe

You can combine several Columns (smart raster levels) as one with "Over" node, or collapsing them, and edit them with the Palette filter, but not individually. For example if I have two columns, A and B, and I want to keep the color #3 of Column A, Palette filter will keep color #3 of Column A and B too

### Describe the solution you'd like

The possibility to choose the color depending on the column.

**Comment by RodneyBaker** [comment]
We'd likely need to know more of the specific need but I'm pretty confident we can get the results you want.
It seems to me the first place to consider would be where you are placing your Palette Filter node.
If it is after the two flows of Column A and B combine (say after the Over node or after the Xsheet node then we are telling the Schematic to treat them as one node.  

Instead we might place the Palette Filter before the node that combines the columns thereby achieving different results.

Here's an example...
Two columns both with the same level/image (just spread across columns)...
Placement of the Palette Filter occurs before the Over node...
Removing the desired color from one instance of the image but not the other. (in this case the color red)

![image](https://user-images.githubusercontent.com/2576432/205190327-8037aedb-d3a3-4556-8ab7-eb469fee0f8e.png)

**Comment by F-Burning** [comment]
My idea is that it works after the over. I also usually collapse several columns into one, and then apply the palette filter to it, and it works, but the problem is that it doesn't recognize the colors of each column individually.

If for example I collapse 5 columns and apply the palette filter, I can't remove a specific color, instead all colors that share the same index will be removed.
